systemd/crc-cluster-status.service

```diff
@@ -13,6 +13,9 @@ Restart=on-failure
 RestartSec=40
 ExecCondition=/usr/local/bin/crc-check-cloud-env.sh
 ExecStart=/usr/local/bin/crc-cluster-status.sh
+ExecStartPost=/usr/bin/rm -rf /opt/crc/pull-secret
+ExecStartPost=/usr/bin/rm -rf /opt/crc/pass_developer
+ExecStartPost=/usr/bin/rm -rf /opt/crc/pass_kubeadmin
```

Comment:
why are removing those so that stop start it doesn't start those service?

Reply:
if we remove those and then user start the cloud instance first time it pass the cloud-init file which will put all those files like pull-secret, pass_developer ..etc to right place. But then user done using this instance and stop it using cloud cli or from web. If it is again start this instance and this time there is no cloud-init run doesn't it make the services fails ?

Reply:
Maybe make the service conditional on the existence of these files so that systemd does not happen to start the service if these are missing?

Reply:
while testing i found that using `ConditionPathExists` doesn't respect `Restart=on-failure`, and in our case the units always has to restart several times before those files are created by `cloud-init` , so when the service starts before cloud-init and the files are not yet present it'll not be restarted again 

(maybe we can schedule the self sufficient services that needs the files to start after `cloud-init-finale.service` and get around this issue)

Reply:
yes other than removing them because they are sensitive, there is no other reason,  i can drop this commit!